phpcs.xml.dist

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
+<ruleset name="Commerce Components Coding Standards">
+
+    <rule ref="WordPress-VIP-Go"/>
+
+    <rule ref="WordPress-Docs">
+        <exclude-pattern>*/tests/*</exclude-pattern>
+    </rule>
+
+    <rule ref="WordPress">
+        <exclude name="WordPress.Files.FileName"/>
+        <exclude name="Generic.Arrays.DisallowShortArraySyntax"/>
+    </rule>
+
+    <rule ref="StellarWP"/>
+
+    <!--exclude the rule for violation of direct DB calls as some have no alternative-->
+    <rule ref="WordPress.DB.DirectDatabaseQuery">
+        <exclude-pattern>src/Test.php</exclude-pattern>
+    </rule>
+
+    <exclude-pattern>*/tests/_support/_generated/*</exclude-pattern>
+    <exclude-pattern>*/vendor/*</exclude-pattern>
```

Comment:
Also, I don't think this file is correct, missing the directories/files to check, e.g.

Otherwise running `phpcs` on its own will always ask for a file/directory.

```xml
    <!-- Directories to be checked -->
    <file>./src</file>
 ````

Reply:
yeah, I ended up breaking out all the coding standards stuff into a new [feature/coding-standards](https://github.com/stellarwp/templates/tree/feature/coding-standards) branch to keep it from muddying up these test updates.